libcob/intrinsic.c

```diff
@@ -1894,7 +1894,7 @@ cob_intr_numval (cob_field *srcfield)
 		make_field_entry (&field);
 		memcpy (curr_field->data, (char *)&llval, 8);
 	} else {
-		snprintf ((char *)final_buff, 63, "%s%s.%s", sign ? "-" : "",
+		snprintf ((char *)final_buff, 130, "%s%s.%s", sign ? "-" : "",
```

Comment:
Even if the second argument is 129, the warning messages will be solved.
Set the second argument to 129.